requirements.txt

```diff
@@ -1,4 +1,5 @@
-ethereum
 click
-coincurve
-web3>=4.0.0-beta.4
+rlp<1.0.0
+coincurve>=7.1.0,<8.0
+ethereum>=2.3.1,<3.0
```

Comment:
is this still used in the tests? or just necessary for the stuff that is fixed in #57?

Reply:
I've used versions from `raiden-libs` and I am happy the tests pass finally :) 

We can probably rebase #57 on top of this PR when it gets to master and update the versions accordingly.

Reply:
Hehe, I get that. Updating it in #57 is fine for me.